playbooks/roles/compatibility-layer/defaults/main.yml

```
@@ -4,6 +4,8 @@ custom_overlay_url: https://github.com/EESSI/gentoo-overlay.git
 
 prefix_location: /cvmfs/pilot.eessi-hpc.org/test/gentoo/2020
 
+pkg_set: '@2020'
```

Would it make sense to make this a list of sets, so that we can support/install multiple sets?
```suggestion
package_sets: 
  - '@2020'
```